### PR TITLE
Consider running rubocop from a sub-folder

### PR DESCRIPTION
**Pull request by rogercampos**
Allows to work with monorepo, where you run `rubocop-changes` from within a folder inside the git repo

### CODE REVIEW

File: lib/rubocop/changes/check.rb [hunk 1]
```diff
@@ -27,7 +27,7 @@ def line_numbers
       end
 
       def lines_from_diff
-        patch.changed_line_numbers
+        patch.nil? ? lines_from_rubocop : patch.changed_line_numbers
```
Comment (rogercampos) [inline lib/rubocop/changes/check.rb]: this seems unrelated but `patch` was nil in the example I was checking 🤔

Comment (fcsonline) [reply lib/rubocop/changes/check.rb]: We can review it together

File: spec/rubocop/changes/checker_spec.rb [hunk 2]
```diff
@@ -17,6 +17,11 @@
     ).run
   end
 
+  before do
+    allow(Rubocop::Changes::Shell).to receive(:run).with(
+      "git rev-parse --show-toplevel")
```
Comment (fcsonline) [inline spec/rubocop/changes/checker_spec.rb]: Maybe we can add a spec covering this case.

Comment (rogercampos) [reply spec/rubocop/changes/checker_spec.rb]: added

**Pull request merged**